scripts/ngi_aggregate_statistics

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+""" 
+This is a script to create NGI aggregate report.
```

Comment:
Ok. I'll keep the plotting in, and move stuff into `common/project_summary.py`. Let you know when I'm done. :smile: